data.sql

```diff
@@ -162,4 +162,8 @@ FULL JOIN specializations ON specializations.vets_id = vets.id
 WHERE vets.name = 'Maisy Smith' 
 GROUP BY S 
 ORDER BY CNT 
-DESC LIMIT 1;
\ No newline at end of file
+DESC LIMIT 1;
+
+--Performance Improvement--
```

Comment:
## 1. Participant section
![image](https://user-images.githubusercontent.com/93039359/215531953-b92271fd-7465-4a6b-9221-1faff3a463f6.png)
## 2. Commits
![image](https://user-images.githubusercontent.com/93039359/215532085-91a1c269-9a13-4022-a82b-5cf1ea80b8cb.png)

- [ ] As you know that this is a pair-programming activity but I cannot see any commit from your partner's side :eyes: Kindly ask your partner @Zainab-Alzubair to make contributions to this project :+1: or if you are working alone on this project and microverse has given your permission to work alone in this project then kindly attach a screenshot of that permission email in your pull request summary to make next reviewer sure that microverse has given you permission to he won't block a valid review on your pull request :+1: